android/quest/src/main/java/org/smartregister/fhircore/quest/ui/launcher/GeoWidgetLauncherViewModel.kt

```diff
@@ -145,4 +175,19 @@ constructor(
             )
         }
     }
+    //todo make our own events eg GeoWidgetEvent
+    fun onEvent(event: RegisterEvent) =
+        when (event) {
```

Comment:
Why do we need to react to events from the Register?

Reply:
updated it on https://github.com/opensrp/fhircore/pull/3148